cita@0.23.rb

```diff
@@ -0,0 +1,33 @@
+# coding: utf-8
+# CITA Formula
+class CitaAT023 < Formula
+  desc "A high performance blockchain for enterprise users."
+  homepage "https://github.com/cryptape/cita"
+
+  version "0.23.1"
```

Comment:
Is it better to install `0.23.0` when our user:

```shell
brew install cita@0.23
```

?

Reply:
i don't think so.

We can just focus the minor version and use the latest patch version:

cita@0.23:  the version of CITA is 0.23.x (x will be the latest pathc version)

Reply:
This is also very reasonable!